### ISSUE TITLE
downloadable?

### ISSUE THREAD

**Issue by luminosity-d3v** [rhenryw/UV-Static-2.0#11]
I want to know if the proxy website can be downloaded so it cannot be locked by viruses and the like. Please respond if it is; or add the further accessibility

**Comment by Airbus-A330-bruh** [comment]
> I want to know if the proxy website can be downloaded so it cannot be locked by viruses and the like. Please respond if it is; or add the further accessibility

its not. if u want one that can run in an html file look at sandstone or https://github.com/nautilus-os/GUST

**Comment by luminosity-d3v** [comment]
ok, thanks for saying so
